checkpoint/cp-resource-management/templates/addons/image-uploader/role.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if .Values.rbac.pspEnable }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Release.Name }}-image-uploader
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ .Release.Name }}-image-uploader
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+rules:
+  - apiGroups: [ "policy", "extensions" ]

```

Comment:
if you remove "if" from podsecuritypolicy.yaml regarding "extensions" - remove "extensions" here